build/dependencies.yaml

```diff
@@ -1,7 +1,7 @@
 dependencies:
   # agnhost: bump this one first
   - name: "agnhost"
-    version: "2.27"
+    version: "2.28"
```

Comment:
something is wrong, seems current published agnhost image is 2.28

 gcr.io/k8s-staging-e2e-test-images/agnhost@sha256:4f3eec1e602e1e0b0e5858c8f3d399d0ef536799ec1f7666e025f211001cb706

Reply:
cc @claudiubelu 
https://github.com/kubernetes/kubernetes/blob/033d950152dc83fcd9afb546f8c112feb9e2c6f7/build/dependencies.yaml#L4-L13